apps/condo/domains/billing/gql.js

```diff
@@ -35,9 +35,12 @@ const BillingAccountMeter = generateGqlQueries('BillingAccountMeter', BILLING_AC
 const BILLING_ACCOUNT_METER_READING_FIELDS = `{ context ${BILLING_INTEGRATION_ORGANIZATION_CONTEXT_FIELDS} importId property { id } account { id } meter { id } period value1 value2 value3 date raw ${COMMON_FIELDS} }`
 const BillingAccountMeterReading = generateGqlQueries('BillingAccountMeterReading', BILLING_ACCOUNT_METER_READING_FIELDS)
 
-const BILLING_RECEIPT_FIELDS = `{ context ${BILLING_INTEGRATION_ORGANIZATION_CONTEXT_FIELDS} importId property { id } account { id } period raw toPay services meta ${COMMON_FIELDS} }`
+const BILLING_RECEIPT_FIELDS = `{ context ${BILLING_INTEGRATION_ORGANIZATION_CONTEXT_FIELDS} importId property { id } account { id } recipient { id } period raw toPay toPayDetails services ${COMMON_FIELDS} }`
```

Comment:
printableNumber?

Reply:
and in readme in BillingOrganization missed context

Reply:
yep, forgot it. Adding....